proof/refine/ARM/CNodeInv_R.thy

```diff
@@ -5481,6 +5481,10 @@ lemma make_zombie_invs':
                       (\<forall>p \<in> threadCapRefs (cteCap cte).
                             st_tcb_at' ((=) Inactive) p s
                              \<and> bound_tcb_at' ((=) None) p s
+                             \<and> obj_at' (\<lambda>tcb. tcbSchedContext tcb = None \<or> tcbSchedContext tcb = Some idle_sc_ptr) p s
```

Comment:
We have `bound_sc_tcb_at’` and `bound_yt_tcb_at’` for these, just like `bound_tcb_at’` above.